File: source/csharp/GameController.cs
using UnityEngine;
using Entitas;
using Entitas.Unity.VisualDebugging;

public class GameController : MonoBehaviour {

    Systems _systems;

    void Start() {
        Random.seed = 42;

        _systems = createSystems(Pools.core);
        _systems.Initialize();
    }

    void Update() {
        _systems.Execute();
    }

    Systems createSystems(Pool pool) {
        #if (UNITY_EDITOR)
        return new DebugSystems()
        #else
        return new Systems()
        #endif

            // Initialize
            .Add(pool.CreateSystem<CreatePlayerSystem>())
            .Add(pool.CreateSystem<CreateOpponentsSystem>())
            .Add(pool.CreateSystem<CreateFinishLineSystem>())

            // Input
            .Add(pool.CreateSystem<InputSystem>())

            // Update
            .Add(pool.CreateSystem<AccelerateSystem>())
            .Add(pool.CreateSystem<MoveSystem>())
            .Add(pool.CreateSystem<ReachedFinishSystem>())

            // Render
            .Add(pool.CreateSystem<RemoveViewSystem>())
            .Add(pool.CreateSystem<AddViewSystem>())
            .Add(pool.CreateSystem<RenderPositionSystem>())

            // Destroy
            .Add(pool.CreateSystem<DestroySystem>());
    }
}

File: example/src/core/GameController.ts
module example {

  import Pools = example.Pools;
  import Systems = entitas.Systems;

  export class GameController {

    systems:Systems;

    start() {

      this.systems = this.createSystems(Pools.core);
      this.systems.initialize();

    }

    update(delta:number) {
      this.systems.execute();
    }

    createSystems(pool) {
      return new Systems()
        // Initialize
        .add(pool.createSystem(example.CreateFinishLineSystem))
        .add(pool.createSystem(example.CreatePlayerSystem))
        .add(pool.createSystem(example.CreateOpponentsSystem))

        // Input
        .add(pool.createSystem(example.InputSystem))

        // Update
        .add(pool.createSystem(example.AccelerateSystem))
        .add(pool.createSystem(example.MoveSystem))
        .add(pool.createSystem(example.ReachedFinishSystem))

        // Render
        .add(pool.createSystem(example.RemoveViewSystem))
        .add(pool.createSystem(example.AddViewSystem))
        .add(pool.createSystem(example.RenderPositionSystem))

        // Destroy
        .add(pool.createSystem(example.DestroySystem));

    }
  }
}